Server/Utilities/CBXDevice.m

```diff
@@ -286,7 +286,13 @@ - (NSDictionary *)formFactorMap {
       @"iPhone14,8" : @"iPhone 14 Plus",
       @"iPhone15,2" : @"iPhone 14 Pro",
       @"iPhone15,3" : @"iPhone 14 Pro Max",
-      
+
+      // iPhone 15 Series.
+      @"iPhone15,4" : @"iPhone 15",
+      @"iPhone15,5" : @"iPhone 15 Plus",
+      @"iPhone16,1" : @"iPhone 15 Pro",
```

Comment:
this is iphone 16 series

Reply:
No, this is iPhone 15. I don't know why Apple decided to go like that. iPhone16,1 is iPhone 15 Pro. I suppose iPhone 16 line up will start from iPhone 16,4